harness/determined/common/constants.py

```diff
@@ -17,6 +17,11 @@
 # We subtract one megabyte to account for any message envelope size we may have.
 MAX_CONTEXT_SIZE = MAX_ENCODED_SIZE - (1 * 1024 * 1024)
 
+# The maximum size of a workload metrics object is capped at 500MB. Metrics
+# are specified by the model definition and persisted after each workload
+# training step is completed.
+MAX_METRICS_SIZE = 500 * (1 << 20)
```

Comment:
Lets change this to 100 MB, unless Hoang told you otherwise?

Reply:
my bad, forgot to change it. done.